src/components/Positions.jsx

```diff
@@ -638,15 +538,9 @@ const PositionInfo = styled.div`
   border-radius: 10px;
   font-size: 18px;
   padding: 40px 0px 40px;
-  @media (${device.tablet}) {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    padding: 4px 10px 4px 10px;
-    max-width: 432px;
-  }
+  width: 100%;
   ${(props) => props.mobile && css`
-  display: flex;
+    display: flex;
```

Comment:
Seems like there's potential for some overlap issues on mobile 
![image](https://user-images.githubusercontent.com/43038510/215506417-b257a412-f585-4c71-82bb-3b350ad44884.png)

Reply:
Desktop view looks fine tho. FYI My node-consensus branch is going to add a button above the row of info boxes.

Reply:
Seemed to work when I tried to replicate just now. Resolving